algebird-core/src/main/scala/com/twitter/algebird/DecayedValue.scala

```diff
@@ -73,4 +73,19 @@ case class DecayedValue(value: Double, scaledTime: Double) extends Ordered[Decay
     val normalization = halfLife / math.log(2)
     value / normalization
   }
+
+  /*
+  * Moving average assuming the signal started at zero a fixed point in the past.
+  * This normalizes by the integral of exp(-t(ln(2))/halflife) from 0 to (endTime - startTime).
+  */
+  def averageFrom(halfLife: Double, startTime: Double, endTime: Double) = {
```

Comment:
(re being suspicious of the math - I definitely come up with the same answer when I do it from first principles, but I struggle with the intuition of an average being higher than any values it's seen so far; I'm sure this is something to do with discrete vs. continuous...)